CORE/utilities/Timer/AndroidTimer.cpp

```diff
@@ -14,7 +14,7 @@ void AndroidTimer::update() {
 
     timespec lTimeVal;
 
-#ifdef __EMSCRIPTEN__
+#if defined(__EMSCRIPTEN__) || defined (__ANDROID__)
```

Comment:
this is not entirely right MONOTONIC will work good on Emulator but not on actual device.